samples/highcharts/series-gauge/speedometer-dual-axes/demo.details

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
We already have this demo as `highcharts/demo/gauge-dual`, so it can be deleted and the docs be directed to the pre-existing one.